amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseRefresher.java

```diff
@@ -30,16 +30,13 @@
     /**
      * Creates the table that will store leases. Succeeds if table already exists.
      * 
-     * @param readCapacity
-     * @param writeCapacity
-     * 
      * @return true if we created a new table (table didn't exist before)
      * 
      * @throws ProvisionedThroughputException if we cannot create the lease table due to per-AWS-account capacity
      *         restrictions.
      * @throws DependencyException if DynamoDB createTable fails in an unexpected way
      */
-    boolean createLeaseTableIfNotExists(Long readCapacity, Long writeCapacity)
```

Comment:
This is a public interface customers might be using; let's not mess with these signatures. We can make `createLeaseTableIfNotExists()` a separate signature to preserve backwards-compatible behavior.

Reply:
ya, kept this and marked as deprecated.